spec/person_spec.rb

```diff
@@ -0,0 +1,42 @@
+require './lib/person.rb'
+require './lib/library.rb'
+
+describe Person do
+  books = [{title: 'Managment101', author: 'Rich'}, {title: 'Cooking101', author: 'Fat'}]
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)